scenarios/kubernetes_bazel.py

```diff
@@ -37,13 +37,31 @@ def check_output(*cmd):
     print >>sys.stderr, 'Run:', cmd
     return subprocess.check_output(cmd)
 
+def find_bazel():
```

Comment:
imo we should stop using this script to invoke bazel directly and instead use some well defined target in k/k
can we manipulate the path instead? (maybe symlink .. or see if we can get bazelisk to pre-download these versions and use bazelisk then..?)

Reply:
I added pre-downloading with bazelisk. Looking at the code, it's going to be hard to use bazelisk in a way that falls back to a version we like instead of trying to download and use latest.

> we should stop using this script to invoke bazel directly and instead use some well defined target in k/k

Would this require extensive changes? Is there anything minimal we can do here to bump the bazel version?

Reply:
Are you familiar with `BAZEL_VERSION=real`? See https://github.com/kubernetes/test-infra/pull/16255

Reply:
Awesome, did not know about the wrapper. Fixed.